ODIN_II/SRC/verilog_flex.l

```diff
@@ -340,7 +340,6 @@ vPUNCT [\?\:\|\^\&\<\>\-\*\/\%\(\)\{\}\[\]\~\!\;\#\,\.\@\=\+]
 <INITIAL>"trior"                     { MP(); return ieee_filter(vTRIOR); }
 <INITIAL>"trireg"                    { MP(); return ieee_filter(vTRIREG); }
 <INITIAL>"uwire"                     { MP(); return ieee_filter(vUWIRE); }
-<INITIAL>"uwire"                     { MP(); return ieee_filter(vUWIRE); }
```

Comment:
@emacdo12 this will cause merge conflict with your PR, but it wouldn't compile without